plugins/catalog/index.js

```diff
@@ -0,0 +1,21 @@
+import * as inversify from "inversify";
+import { RoutingServiceExtension, NavigationServiceExtension } from "dcos-sdk";
+
+import RepositoriesRouting from "./src/js/repositories/RepositoriesRouting";
+import CatalogRouting from "./src/js/catalog/CatalogRouting";
+import RepositoriesNavigation
+  from "./src/js/repositories/RepositoriesNavigation";
+import CatalogNavigation from "./src/js/catalog/CatalogNavigation";
+
+// Common interface. It will be used to dynamically parameterize modules.
```

Comment:
This comment should not be there in production

Reply:
Aren't we removing comments during build?

Reply:
```diff
- This comment should not be there in production
+ This comment should not be there on master
```